apps/src/music/views/Callouts.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {useSelector} from 'react-redux';
+import {MusicState} from '../redux/musicRedux';
+import moduleStyles from './callouts.module.scss';
+const FontAwesome = require('../../templates/FontAwesome');
+
+const availableCallouts: {
+  [key: string]: {selector: string; arrowAngle: number};
+} = {
+  'play-sound-block': {
+    selector: ".blocklyFlyout g[data-id='play-sound-block']",
+    arrowAngle: 0,
+  },
+  'when-run-block': {
+    selector: "g[data-id='when-run-block'] > path",
+    arrowAngle: 0,
+  },
+  'run-button': {selector: '#run-button', arrowAngle: 180},
+};
+
+/**
+ * Renders one of several pre-defined callouts.
+ */
+const Callouts: React.FunctionComponent = () => {
+  const callout = useSelector(
+    (state: {music: MusicState}) => state.music.showCallout
+  );
+
+  if (callout.id) {
+    const availableCallout = availableCallouts[callout.id];
```

Comment:
Need to handle the case where we don't have an available registered callout for the given ID?

Reply:
Good call.  Fixed.